test/core/end2end/end2end_tests.h

```diff
@@ -324,7 +324,7 @@ class CoreEnd2endTest : public ::testing::Test {
     grpc_op MakeOp();
 
    private:
-    int cancelled_;
```

Comment:
The expectation here is that this is set by core prior to the operation completing, so I think we're just masking bugs here?